sormas-backend/src/main/java/de/symeda/sormas/backend/visit/VisitService.java

```diff
@@ -144,8 +208,18 @@ public Predicate buildRelevantVisitsFilter(Person person, Disease disease, Date
 		filter = and(
 			cb,
 			filter,
-			cb.greaterThanOrEqualTo(from.get(Visit.VISIT_DATE_TIME), DateHelper.subtractDays(startDate, ContactLogic.ALLOWED_CONTACT_DATE_OFFSET)),
-			cb.lessThanOrEqualTo(from.get(Visit.VISIT_DATE_TIME), DateHelper.addDays(endDate, ContactLogic.ALLOWED_CONTACT_DATE_OFFSET)));
+			or(
```

Comment:
This code doesn't really make sense to me because it doesn't differentiate between cases or contacts. If CaseLogic.ALLOWED_CASE_DATE_OFFSET differs from ContactLogic.ALLOWED_CONTACT_DATE_OFFSET, both options will be considered for both cases and contacts which means that this filter won't work properly. Only one offset should be used depending on whether visits are queried for cases or contacts

Reply:
OK to merge CaseLogic.ALLOWED_CASE_DATE_OFFSET and ContactLogic.ALLOWED_CONTACT_DATE_OFFSET (and VisitDto.ALLOWED_CONTACT_DATE_OFFSET?)
It goes in the direction of jointly managing the logics of the visits.

Reply:
I think that should be fine, yes.